Cargo.toml

```diff
@@ -214,7 +214,10 @@ feat_os_windows_legacy = [
 [workspace]
 
 [dependencies]
-uucore   = "0.0.2"
+lazy_static = { version="1.3" }
+textwrap = { version="0.11.0", features=["term_size"] }
```

Comment:
`textwrap` is unstable, with "term_size" being an undocumented required features which is now updated to something different in the repo ... I'm actually thinking this might need to be pinned to this specific version.

Reply:
I was going to wait an pin it if/when we get a compiler error. Or should I do it now with a comment?